app/controllers/openid_application_controller.rb

```diff
@@ -125,8 +133,8 @@ def fetch_profile(uid)
     end
   end
 
-  def build_session(ttl)
-    session = Session.new(token: token.to_s, uuid: token.identifiers.uuid)
+  def build_session(ttl, profile)
```

Comment:
Rather than storing the entire Okta profile response here as an object, I think we should minimize the profile down to only the necessary information.

That should only be:
- last_login_type
- icn
- sec_id
- vista_id
- npi